Sources/Penny/+Array.swift

```diff
@@ -28,6 +28,12 @@ extension Array {
 }
 
 extension Array<String> {
+    mutating func appendUnique(_ element: String) {
```

Comment:
This is just _screaming_ out that what you really want here is `swift-collections`'s `OrderedSet`

Reply:
Hmmm good idea